include/flexflow/ops/element_binary.h

```diff
@@ -24,12 +35,18 @@ class ElementBinaryMeta : public OpMeta {
 
 class ElementBinary : public Op {
 public:
+  using Params = ElementBinaryParams;
+
   ElementBinary(FFModel& model,
                 OperatorType type,
                 const ParallelTensor x,
                 const ParallelTensor y,
                 bool inplace_a,
                 const char* name);
+  ElementBinary(FFModel& model,
+                const Params& params,
+                const std::vector<ParallelTensor>& inputs,
```

Comment:
Is there a reason for the `vector` here instead of either two `ParallelTensor` or a `std::pair<ParallelTensor>` since we know the number of inputs should always be two?